doc/modules/ROOT/pages/graph-object.adoc

```diff
@@ -196,6 +196,27 @@ n = G.node_count()
 props = G.node_properties("City")
 ----
 
+== Context managament 
+
+The graph object also implement the context managment protocol, i.e., is usable inside `with` cluases.
+This allows automatically dropping the graph at the end of `with` block.
```

Comment:
Nitpick: I think a bit better to say something like: "When used this way, the graph projection will be automatically dropped on the server side when exiting the `with` block.